packages/twenty-server/src/workspace/messaging/commands/crons/fetch-all-workspaces-messages.cron.pattern.ts

```diff
@@ -1 +1 @@
-export const fetchAllWorkspacesMessagesCronPattern = '*/10 * * * *';
+export const fetchAllWorkspacesMessagesCronPattern = '*/1 * * * *';
```

Comment:
@Weiko Is it intended ?

Reply:
@magrinj Oops no! I wanted to do some tests locally, not intended